modules/meb_api/spec/requests/meb_api/v0/forms_spec.rb

```diff
@@ -121,4 +131,178 @@
+          expect(response.headers['Content-Type']).to eq('application/pdf')
+          expect(response.headers['Content-Disposition']).to include('Post-9%2F11 GI_Bill_CoE')
+        end
+      end
+    end
+
+    context 'when claimant is not eligible' do
+      let(:claim_status_response) { double('claim_status_response', claim_status: 'DENIED') }
+
+      it 'returns a PDF with denial filename' do
+        travel_to Time.zone.local(2024, 1, 15, 10, 30, 0) do
+          get '/meb_api/v0/forms_claim_letter', params: { type: 'ToeSubmission' }
+          expect(response).to have_http_status(:ok)
+          expect(response.headers['Content-Type']).to eq('application/pdf')
+          expect(response.headers['Content-Disposition']).to include('Post-9%2F11 GI_Bill_Denial')
+        end
+      end
+    end
+
+    context 'when claimant response is invalid' do
+      let(:claimant_response) { double('claimant_response', :[] => nil, status: 404, body: 'Error content') }
+
+      it 'returns claimant error response' do
+        get '/meb_api/v0/forms_claim_letter', params: { type: 'ToeSubmission' }
+        expect(response).to have_http_status(:ok)
+        expect(response.body).to eq('Error content')
+      end
+    end
+  end
+
+  describe 'POST /meb_api/v0/forms_submit_claim' do
+    let(:submission_service) { instance_double(MebApi::DGI::Forms::Submission::Service) }
+    let(:submission_response) { double('submission_response', status: 200) }
+
+    before do
+      allow(MebApi::DGI::Forms::Submission::Service).to receive(:new).and_return(submission_service)
+      allow(submission_service).to receive(:submit_claim).and_return(submission_response)
+    end
+
+    context 'when submitting a claim without direct deposit feature' do
+      it 'successfully submits the claim' do
+        post '/meb_api/v0/forms_submit_claim', params: { test_param: 'value' }
+        expect(response).to have_http_status(:ok)
+        expect(JSON.parse(response.body)['data']['status']).to eq(200)
+      end
+    end
+
+    context 'when submitting with form_id to clear saved form' do
+      it 'clears the saved form after submission' do
+        expect_any_instance_of(MebApi::V0::FormsController).to receive(:clear_saved_form).with('12345')
+        post '/meb_api/v0/forms_submit_claim', params: { form_id: '12345' }
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context 'when direct deposit feature is enabled' do
+      let(:direct_deposit_client) { instance_double(DirectDeposit::Client) }
+      let(:payment_info) { { account_number: '1234', routing_number: '5678' } }
+
+      before do
+        Flipper.enable(:toe_light_house_dgi_direct_deposit)
+        allow(DirectDeposit::Client).to receive(:new).and_return(direct_deposit_client)
+        allow(direct_deposit_client).to receive(:get_payment_info).and_return(payment_info)
+      end
+
+      after do
+        Flipper.disable(:toe_light_house_dgi_direct_deposit)
```

Comment:
Again, please mock flippers, `allow(Flipper).to receive(:enabled?).with(:feature_flag).and_return(true/false)`